cypress/integration/data-publication/AggregateReports.spec.js

```diff
@@ -1,88 +1,166 @@
-import { openSelector } from '../../support/helpers'
+import { openSelector } from "../../support/helpers"
```

Comment:
quote changes are from eslint?

Reply:
Looks like it, that change is from @Michaeldremy.  We're working on unifying code conventions in https://github.com/cfpb/hmda-frontend/issues/1440.